beacon_listener:
    scan_time: 1
    max_beacon_age: 1
    publish_rate: 1
    group_id: 0x11111111
